build-system/tasks/bundle-size/index.js

```diff
@@ -115,8 +124,7 @@ async function storeBundleSize() {
   const commitHash = gitCommitHash();
   log('Storing bundle sizes for commit', cyan(shortSha(commitHash)) + '...');
   try {
-    const requestPost = util.promisify(require('request').post); // Lazy-required to speed up task loading.
-    const response = await requestPost({
+    const response = await getRequestPost()({
```

Comment:
Pretty sure this requires a double `await`

Reply:
Yep. I did plan on checking the CI logs before merging this :)

![image](https://user-images.githubusercontent.com/26553114/111835580-84a3d480-88cb-11eb-88f6-a22486cbdf43.png)

Reply:
Fixed:
![image](https://user-images.githubusercontent.com/26553114/111840315-9472e700-88d2-11eb-95ba-e4c38586ff81.png)